src/main/resources/entity-types/finance/simple_fund.json5

```
@@ -23,38 +29,119 @@
       sourceAlias: 'fund',
       queryable: true,
       visibleByDefault: false,
+      essential: true,
       valueGetter: "(:sourceAlias.jsonb ->> '_version')::integer",
       valueFunction: '(:value)::integer',
     },
     {
       name: 'allocated_from_ids',
       dataType: {
-        dataType: 'arrayType',
+        dataType: 'jsonbArrayType',
         itemDataType: {
           dataType: 'rangedUUIDType',
         },
       },
       sourceAlias: 'fund',
       queryable: true,
       visibleByDefault: false,
-      valueGetter: "(SELECT array_agg(alc_id.value::text) FILTER (WHERE (alc_id.value::text) IS NOT NULL) AS array_agg FROM jsonb_array_elements_text(:sourceAlias.jsonb -> 'allocatedFromIds'::text) alc_id(value))",
-      filterValueGetter: "( SELECT array_agg(lower(alc_id.value::text)) FILTER (WHERE (alc_id.value::text) IS NOT NULL) AS array_agg FROM jsonb_array_elements_text(:sourceAlias.jsonb -> 'allocatedFromIds'::text) alc_id(value))",
+      essential: true,
+      hidden: true,
+      valueGetter: "(:sourceAlias.jsonb -> 'allocatedFromIds'::text)",
+    },
+    {
+      name: 'allocated_from',
+      dataType: {
+        dataType: 'jsonbArrayType',
+        itemDataType: {
+          dataType: 'stringType'
+        }
+      },
+      essential: true,
+      queryable: true,
+      visibleByDefault: true,
+      sourceAlias: 'fund',
+      valueGetter: "COALESCE(( \
+                      SELECT jsonb_agg(fund_name) \
+                      FROM ( \
+                            SELECT jsonb ->> 'name' AS fund_name \
+                            FROM ${tenant_id}_mod_fqm_manager.src_finance_fund \
+                            WHERE id IN (SELECT jsonb_array_elements_text(:sourceAlias.jsonb -> 'allocatedFromIds')::uuid) \
+                            ) sub \
+                    ), '[]'::jsonb)",
       valueFunction: 'lower(:value)',
+      filterValueGetter: "COALESCE(( \
+                            SELECT jsonb_agg(fund_name) \
```

Since the `valueFunction` does a `lower()`, shouldn't this be something like this?
```suggestion
                            SELECT jsonb_agg(lower(fund_name)) \
```